angular-ngrx-scss/src/app/home/profile/profile-about/profile-about.component.html

```diff
@@ -0,0 +1,79 @@
+<ng-container *ngIf="profile$ | async as profile">
+  <img
+    *ngIf="profile?.user?.avatar"
+    class="avatar"
+    [src]="profile?.user?.avatar"
+  />
+
+  <div class="content">
+    <h1>
+      <div class="name">{{ profile?.user?.name }}</div>
+      <div class="username">{{ profile?.user?.username }}</div>
+    </h1>
+    <div class="bio">{{ profile?.user?.bio }}</div>
+    <div class="socials">
+      <ng-container *ngFor="let social of socials; let i = index">
+        <span
+          *ngIf="social.icon"
+          class="icon"
+          appOcticon="{{ social.icon }}"
+          color="black"
+          size="16"
+        ></span>
+        <span class="social-label">{{ social.value }} {{ social.label }}</span>
+        <!-- Separator dot between social entries. -->
+        {{ i === socials.length - 1 ? '' : '·' }}
+      </ng-container>
+    </div>
+    <div class="fields">
+      <div *ngIf="profile?.user?.company">
+        <span class="icon" appOcticon="organization"></span>
+        <span>{{ profile?.user?.company }}</span>
+      </div>
+      <div *ngIf="profile?.user?.location">
+        <span class="icon" appOcticon="location"></span
+        ><span>{{ profile?.user?.location }}</span>
+      </div>
+      <div *ngIf="profile?.user?.email">
+        <span class="icon" appOcticon="mail"></span>
+        <a href="mailto:{{ profile?.user?.email }}">{{
+          profile?.user?.email
+        }}</a>
+      </div>
+      <div *ngIf="profile?.user?.blog">
+        <span class="icon" appOcticon="link"></span>
+        <a [href]="profile?.user?.blog">{{ profile?.user?.blog }}</a>
+      </div>
+      <div *ngIf="profile?.user?.twitter_username">
+        <span class="icon twitter-icon">
+          <svg
```

Comment:
This seems fairly small, but if the octicons package doesn't have a Twitter icon, would like to have this as a file in the assets folder that we pull in here instead of the direct svg, unless it's massively more complex to do that way.

Reply:
I embedded to avoid an extra network call for the icon. I wonder if we can get the best of both worlds somehow?